src/components/UI/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { mixin } from "../../../styles";
+
+interface TextareaProps {
+  name: string;
+  placeholder?: string;
+  required?: boolean;
+}
+
+export const Textarea = ({ ...props }: TextareaProps) => {
+  return <textarea {...props} />;
+};
+
+export const TextareaDefault = styled(Textarea)`
```

Comment:
you should be able to replace `styled(Textarea)` by `styled.textarea`

Reply:
yup ok